recipes/dcmstack/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "v0.6.2" %}
+
+package:
+  name: dcmstack
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/moloney/dcmstack.git
+  git_tag: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - dcmstack
+  commands:
+    - python -c "import example;"
```

Comment:
Ha! Yes - good catch. As you might expect, I am using `example` as a template for this (as a side note, would be good to have some comments in that one explaining what each part does!). What do you mean "part of the regular import tests?"